.github/workflows/main.yml

```diff
@@ -12,9 +12,8 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        name: [
-          ubuntu-clang-iwyu,
-        ]
+        name: 
+          - ubuntu-clang-iwyu
```

Comment:
For the merge conflict you can just remove the `name:` here, the `include:` already takes care of everything.